addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1316,22 +1314,13 @@ def _create_sub_channel(self, from_message_id=None, name=None):
     @api.readonly
     @api.model
     def get_mention_suggestions(self, search, limit=8):
-        """ Return 'limit'-first channels' id, name, channel_type and authorizedGroupFullName fields such that the
+        """ Return 'limit'-first channels' id, name, channel_type and group_public_id fields such that the
             name matches a 'search' string. Exclude channels of type chat (DM) and group.
         """
         domain = [("name", "ilike", search), ("channel_type", "=", "channel")]
         channels = self.search(domain, limit=limit)
-        return [{
-            'authorizedGroupFullName': channel.group_public_id.full_name,
-            'channel_type': channel.channel_type,
-            'model': "discuss.channel",
-            'id': channel.id,
-            'name': channel.name,
-            'parent_channel_id': {
-                'id': channel.parent_channel_id.id,
-                'model': 'discuss.channel'
-            } if channel.parent_channel_id else False,
-        } for channel in channels]
+        store = Store(channels)
```

Comment:
We might not want to return the full channel store here, but only list the specific fields that were previously used.